src/plugins/stylesheet/LESSPlugin.ts

```diff
@@ -43,7 +43,7 @@ export class LESSPluginClass implements Plugin {
             less = require("less");
         }
 
-        options.filename = file.info.fuseBoxPath;
+        options.filename = file.info.absDir;
```

Comment:
DId you check how well it works with the sourcemaps?
I think we should make it optional.